README.md

```diff
@@ -0,0 +1,422 @@
+<p align="center">
```

Comment:
Describe the technologies and tools used, like Spring Boot, Spring Security, Spring Data JPA, Swagger, etc.